cdisc_rules_engine/services/data_readers/data_reader_factory.py

```diff
@@ -20,6 +24,7 @@ def register_service(cls, name: str, service: Type[DataReaderInterface]):
         """
         Registers a new service in internal _service_map
         """
+        print("registering data reader service with name = ", name)
```

Comment:
no need to register the json data reader because it is already added to the reader_map. Registering services is for external users to extend the tool without having to make a PR